e2e/test-component/scenarios/embedding-sdk/interactive-question-theming.cy.spec.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  InteractiveQuestion,
+  type MetabaseTheme,
+} from "@metabase/embedding-sdk-react";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { createQuestion, describeEE } from "e2e/support/helpers";
+import {
+  mockAuthProviderAndJwtSignIn,
+  mountSdkContent,
+  signInAsAdminAndEnableEmbeddingSdk,
+} from "e2e/support/helpers/component-testing-sdk";
+import { getSdkRoot } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+import { Box } from "metabase/ui";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+const SAMPLE_THEME = {
```

Comment:
A theme that makes full use of the theme customizations for components. I made the theme values different between components so it's easy to identify in the e2e. A visual regression test would work better here, but we don't have that yet so an e2e is needed.

P.S. I tried writing a Jest test, but JSDOM does not support CSS variables so the assertion does not work.